sample-data/update_avro_from_aws.sh

```diff
@@ -15,15 +15,15 @@ echo "INFO: Checking for avro schemas from ${date} in ${bucket}"
 for path in $(aws s3 ls ${bucket}${prefix});
 do
   if [[ $path != "" ]] && [[ $path != "PRE" ]]; then
-    event_type=$(echo $path | sed 's/event_type=//g;s/\///g')
+    event_type=$(echo $path | sed 's/users.//g;s/\///g')
     payloadpath=$(aws s3 ls "${bucket}${prefix}${path}${date}/" --recursive | sort | tail -n 1 | awk '{print $4}')
     if [[ $payloadpath == "" ]]; then
-      echo "WARNING: Looks like we can't find anything for the current date ${date}, check the hour before."
+      echo "WARNING: Looks like we can't find anything for the current date ${date} in path ${path}, check the hour before."
       prev_date="date=$(date -u -v-60M '+%Y-%m-%d-%H')"
       payloadpath=$(aws s3 ls "${bucket}${prefix}${path}${prev_date}/" --recursive | sort | tail -n 1 | awk '{print $4}')
       if [[ $payloadpath == "" ]]; then
-        echo "ERROR: Couldn't find data in ${prev_date} either. Exiting."
-        exit 1
+        echo "ERROR: Couldn't find data in ${prev_date} either. Skipping!"
+        continue
```

Comment:
I have some concern here that this might make it easy to miss in the logs. Can we cache the number of errors and make sure exit code is still 1? Or make the error message louder? Or add an instruction somewhere?

Reply:
We can no longer exit on failed checks because there are other directories in this s3 bucket. I'll try and write a cleaner way around looking at those buckets.

Reply:
Okay fixed this so we only check `users.*` buckets and no longer skip.